### ISSUE TITLE
Cannot compile for IOJS 3+

### ISSUE THREAD

**Issue by kjlaw89** [tojocky/node-printer#80]
Here's the error I get:

```
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  node_printer.cc
  node_printer_win.cc
..\src\node_printer_win.cc(645): error C2039: 'HasIndexedPropertiesInExternalAr
rayData' : is not a member of 'v8::Object' [C:\Users\Kevin\Documents\Work\deskt
op-new\node_modules\printer\build\node_printer.vcxproj]
          c:\users\kevin\.node-gyp\3.0.0\include\node\v8.h(2562) : see declarat
  ion of 'v8::Object'
..\src\node_printer_win.cc(647): error C2039: 'GetIndexedPropertiesExternalArra
yData' : is not a member of 'v8::Object' [C:\Users\Kevin\Documents\Work\desktop
-new\node_modules\printer\build\node_printer.vcxproj]
          c:\users\kevin\.node-gyp\3.0.0\include\node\v8.h(2562) : see declarat
  ion of 'v8::Object'
..\src\node_printer_win.cc(648): error C2039: 'GetIndexedPropertiesExternalArra
yDataLength' : is not a member of 'v8::Object' [C:\Users\Kevin\Documents\Work\d
esktop-new\node_modules\printer\build\node_printer.vcxproj]
          c:\users\kevin\.node-gyp\3.0.0\include\node\v8.h(2562) : see declarat
  ion of 'v8::Object'
```

**Comment by kjlaw89** [comment]
Just to tag onto this, that appears to be the only issue that I've come across so far. If I comment out the few lines that cause the compiling error everything compiles and runs just like I would expect it to.

**Comment by tojocky** [comment]
already fixed in version 0.2.0, please retry to install. open a separate issue if you still have problems.
